package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Deconst control repository build for Strider CD",
   "main": "index.js",
   "scripts": {
+    "format": "standard",
```

Comment:
Hahaha. You disagree?

I switched the content-service and presenter over to real `semistandard` a while ago anyway, because the Atom linter plugin won't pick up `obama`. Sorry, @rgbkrk.